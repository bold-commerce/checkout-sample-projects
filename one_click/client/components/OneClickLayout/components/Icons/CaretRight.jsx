import React from "react";

const CaretRight = () => {
    return (
        <svg width="20" height="20" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
            <g opacity="0.8">
                <path fillRule="evenodd" clipRule="evenodd" d="M9 5.91943L16.6008 12.0001L9 18.0807V5.91943ZM11 10.0807V13.9194L13.3992 12.0001L11 10.0807Z" fill="black"/>
            </g>
        </svg>
    )
}

export default CaretRight;