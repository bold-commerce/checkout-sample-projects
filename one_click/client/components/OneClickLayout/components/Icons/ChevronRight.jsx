import React from "react";

const ChevronRight = () => {
  return(
    <svg width="20" height="20" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
      <path d="M14.5858 12L6.29289 3.70711C5.90237 3.31658 5.90237 2.68342 6.29289 2.29289C6.68342 1.90237 7.31658 1.90237 7.70711 2.29289L16.7071 11.2929C17.0976 11.6834 17.0976 12.3166 16.7071 12.7071L7.70711 21.7071C7.31658 22.0976 6.68342 22.0976 6.29289 21.7071C5.90237 21.3166 5.90237 20.6834 6.29289 20.2929L14.5858 12Z" fill="black"/>
    </svg>
  )
}

export default React.memo(ChevronRight);
