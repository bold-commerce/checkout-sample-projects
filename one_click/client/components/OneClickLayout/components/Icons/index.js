export { default as TimesCircle } from './TimesCirlce'
export { default as ArrowRight } from './ArrowRight'
export { default as CaretRight } from './CaretRight'