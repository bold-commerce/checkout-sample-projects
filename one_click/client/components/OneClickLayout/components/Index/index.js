export { default as IndexPage } from "./IndexPage";
export { default as IndexGuest } from "./IndexGuest";