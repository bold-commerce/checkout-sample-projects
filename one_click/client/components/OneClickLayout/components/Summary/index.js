export { default as Summary } from './Summary';
export { default as SummaryCondensed } from './SummaryCondensed';