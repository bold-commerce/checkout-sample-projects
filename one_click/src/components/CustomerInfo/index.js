export { default as CustomerInfo } from './CustomerInfo'