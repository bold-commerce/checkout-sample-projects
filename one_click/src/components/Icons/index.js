export { default as ChevronRight } from './ChevronRight'
export { default as TimesCircle } from './TimesCircle'
export { default as ArrowRight } from './ArrowRight'
export { default as CaretRight } from './CaretRight'
export { default as Times } from './Times'
export { default as Plus } from './Plus'