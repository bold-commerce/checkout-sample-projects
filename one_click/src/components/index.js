export * from './Address';
export * from './BackButton';
export * from './Billing';
export * from './BillingAddress';
export * from './Card';
export * from './CheckoutButton';
export * from './CheckoutForm';
export * from './Confirmation';
export * from './CustomerInfo';
export * from './Discount';
export * from './EmptyState';
export * from './Header';
export * from './Icons';
export * from './Index';
export * from './InventoryItem';
export * from './LineItems';
export * from './LoadingState';
export * from './Payment';
export * from '../pages/Processing';
export * from './Product';
export * from './RadioItem';
export * from './Shipping';
export * from './ShippingLines';
export * from './Summary';
