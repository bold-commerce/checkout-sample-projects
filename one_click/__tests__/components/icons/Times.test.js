import React from "react";
import { render } from "@testing-library/react";
import { Times } from "../../../src/components/Icons";

describe('Times', () => {
  test('renders Times icon', () => {
    const { asFragment } = render( <Times/> )

    expect(asFragment()).toMatchSnapshot();
  })
});