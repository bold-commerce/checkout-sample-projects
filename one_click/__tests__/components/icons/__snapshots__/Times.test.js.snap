// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Times renders Times icon 1`] = `
<DocumentFragment>
  <svg
    aria-labelledby="button"
    fill="none"
    height="20"
    viewBox="0 0 24 24"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      opacity="0.8"
    >
      <path
        clip-rule="evenodd"
        d="M17.7071 7.70706L7.70712 17.7071L6.29291 16.2928L16.2929 6.29285L17.7071 7.70706Z"
        fill="black"
        fill-rule="evenodd"
      />
      <path
        clip-rule="evenodd"
        d="M16.2929 17.7071L6.29291 7.70706L7.70712 6.29285L17.7071 16.2928L16.2929 17.7071Z"
        fill="black"
        fill-rule="evenodd"
      />
    </g>
  </svg>
</DocumentFragment>
`;
